ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TopToolbarView.swift

```diff
@@ -777,7 +777,8 @@ class TopToolbarView: UIView, ToolbarProtocol {
       urlTextField.text = ""
     }
 
-    DispatchQueue.main.async {
+    DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
+      // have to add some delay to make sure text field is in the view hierarchy
```

Comment:
0.5 seems long, is there any better deterministic way to ensure its ready to become the first responder?